### ISSUE TITLE
Keyboard interaction with tweakers doesn't reset particles

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/rutherford-scattering#127]
From #125.

**Comment by jessegreenberg** [comment]
Thanks for catching this @terracoda, should be fixed in the above commit.

**Comment by terracoda** [comment]
Glad I noticed something after testing the wrong version :-)
